### PR TITLE
Reworked README & supplimenting docs

### PR DESCRIPTION
**Pull request by piercemorris**
Part of [this issue.](https://github.com/themoeway/yomitan/issues/469)

This can be opinionated but essentially I've moved any large sections to its own documentation, and tried to keep the README slim. I've kept content changes minimal and didn't visit the FAQ section so any comments regarding that would be helpful.

Anything else that can be removed/updated/added revolving the docs I can add it into here no problem!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,24 @@
...
 Since the owner requested forks be uniquely named, we have chosen a new name, _yomitan_. (_-tan_ is an honorific used for anthropomorphic moe characters.) While we've made some substantial changes, the majority of the extension's functionality is thanks to hard work of foosoft and numerous other open source contributors from 2016-2023.
 
-Since this is a distributed effort, we **highly welcome new contributors**! Feel free to browse the issue tracker, and you can find us on [TheMoeWay Discord](https://discord.gg/UGNPMDE7zC) at [#yomitan-development](https://discord.com/channels/617136488840429598/1081538711742844980).
-
-## Tool Introduction
+## What's Yomitan?<!-- omit from toc -->
 
 Yomitan turns your web browser into a tool for building Japanese language literacy by helping you to decipher texts
-which would be otherwise too difficult tackle. This extension is similar to
-[10ten Japanese Reader (formerly Rikaichamp)](https://addons.mozilla.org/en-US/firefox/addon/10ten-ja-reader/) for Firefox and
```
Comment (Scrub1492) [inline README.md:33]: Could probably keep an alternatives section somewhere.

File: README.md [hunk 2]
```diff
@@ -49,363 +35,59 @@ Yomitan provides advanced features not available in other browser-based dictiona
...
-  - [Flashcard Creation](#flashcard-creation)
-- [Keyboard Shortcuts](#keyboard-shortcuts)
-- [Advanced Options](#advanced-options)
-  - [Parse sentences using MeCab](#parse-sentences-using-mecab)
-- [Frequently Asked Questions](#frequently-asked-questions)
-- [Licenses](#licenses)
-- [Third-Party Libraries](#third-party-libraries)
+## Documentation<!-- omit from toc -->
+
+- [Migrating from Yomichan](./docs/yomichan-migration.md#migrating-from-yomichan)
```
Comment (Scrub1492) [inline README.md:40]: These links only appear once and the user is likely to miss them. Could probably keep a concise summarized section for each of them, so it flows better (for example, anki integration is a core feature of yomitan but very unnoticeable)

File: README.md [hunk 3]
```diff
@@ -49,363 +35,59 @@ Yomitan provides advanced features not available in other browser-based dictiona
...
+    - The <img src="ext/images/question-mark-circle.svg" width="16" height="16" alt="question mark symbol"> _question mark_ button will open the Information page.
+    - The <img src="ext/images/profile.svg" width="16" height="16" alt="profile icon"> _profile_ button will appear when multiple profiles exist, allowing the current profile to be quickly changed.
 
-Online dictionaries will not be implemented because it is not possible to support them in a robust way. In order to
-perform Japanese deinflection, Yomitan must execute dozens of database queries for every single word. Factoring in
-network latency and the fragility of web scraping, it would not be possible to maintain a good and consistent user
-experience.
+2.  Import the dictionaries you wish to use for term and kanji searches, head over to the [the dictionary docs](./docs/dictionaries.md) to get set up!. If you do not have any dictionaries installed or enabled, Yomitan will warn you that it is not ready for use by displaying an orange exclamation mark over its icon. This exclamation mark will disappear once you have installed and enabled at least one dictionary.
```
Comment (Scrub1492) [inline README.md:74]: change !. to !

File: README.md (hunk 1 above)
Comment (piercemorris) [reply README.md:33]: Added back in

File: README.md (hunk 2 above)
Comment (Scrub1492) [reply README.md]: lgtm

File: README.md [hunk 4]
```diff
@@ -49,363 +35,61 @@ Yomitan provides advanced features not available in other browser-based dictiona
...
-cause the browser to become unresponsive. The time it takes to delete a single dictionary can sometimes be roughly
-the same as the time it originally took to import, which can be significant for certain large dictionaries.
+5.  Click on individual kanji in the term definition results to view additional information about those characters, including stroke order diagrams, readings, meanings, as well as other useful data.
 
-**Why aren't EPWING dictionaries bundled with Yomitan?**
+    <img src="resources/images/search-popup-kanji.png" width="300" alt="popup with kanji details">
 
-The vast majority of EPWING dictionaries are proprietary, so they are unfortunately not able to be included in
-this extension due to copyright reasons.
+To further enhance your Yomitan experience, it's worth [integrating with Anki](./docs/anki-integration.md), a spaced-repetition flashcard program to help solidify the words you encounter
```
Comment (Scrub1492) [inline README.md:88]: missing punctuation

**Comment by piercemorris** [comment]
@djahandarie @toasted-nutbread do these changes look good to you?

File: docs/keyboard-shortcuts.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+| Shortcut                         | Action                                  |
+| -------------------------------- | --------------------------------------- |
+| <kbd>Esc</kbd>                   | Cancel current search.                  |
+| <kbd>Alt</kbd> + <kbd>PgUp</kbd> | Page up through results.                |
+| <kbd>Alt</kbd> + <kbd>PgDn</kbd> | Page down through results.              |
+| <kbd>Alt</kbd> + <kbd>End</kbd>  | Go to last result.                      |
+| <kbd>Alt</kbd> + <kbd>Home</kbd> | Go to first result.                     |
+| <kbd>Alt</kbd> + <kbd>Up</kbd>   | Go to previous result.                  |
+| <kbd>Alt</kbd> + <kbd>Down</kbd> | Go to next result.                      |
+| <kbd>Alt</kbd> + <kbd>b</kbd>    | Go to back to source term.              |
```
Comment (toasted-nutbread) [inline docs/keyboard-shortcuts.md:21]: Capitalize single letter keys.

**Review by toasted-nutbread — COMMENTED** [review]
Can you add newlines to the end of the files that don't have it? A lot of people have settings which auto-format with an empty newline at the end on save which can cause unnecessary git diffs when those are added. (That's probably something that should be a configuration thing for the project settings.)

**Comment by MarvNC** [comment]
> Can you add newlines to the end of the files that don't have it? A lot of people have settings which auto-format with an empty newline at the end on save which can cause unnecessary git diffs when those are added. (That's probably something that should be a configuration thing for the project settings.)

Maybe prettier? It could also make line breaking more consistent if you enable the `proseWrap` option.

**Comment by piercemorris** [comment]
> Can you add newlines to the end of the files that don't have it? A lot of people have settings which auto-format with an empty newline at the end on save which can cause unnecessary git diffs when those are added. (That's probably something that should be a configuration thing for the project settings.)

Adding to what @MarvNC mentioned, I feel like prettier would solve this issue rather than adhoc comments on the PRs - #517. I will make the amendments as you suggested.

File: README.md [hunk 6]
```diff
@@ -49,363 +35,61 @@ Yomitan provides advanced features not available in other browser-based dictiona
...
 
-**Why aren't EPWING dictionaries bundled with Yomitan?**
+    <img src="resources/images/search-popup-kanji.png" width="300" alt="popup with kanji details">
 
-The vast majority of EPWING dictionaries are proprietary, so they are unfortunately not able to be included in
-this extension due to copyright reasons.
+To further enhance your Yomitan experience, it's worth [integrating with Anki](./docs/anki-integration.md), a spaced-repetition flashcard program to help solidify the words you encounter.
 
-**When are you going to add support for $MYLANGUAGE?**
+## Contributing
```
Comment (djahandarie) [inline README.md:90]: I think I'd prefer this to be where it was originally, as this information is high priority (to me). We need to keep a very strong developer on-boarding pipeline for the project to stay alive.

**Review by djahandarie — COMMENTED** [review]
I'm a little skeptical that the subpages will be visible enough for people, I feel like they might get missed when someone is looking for some piece of information. But we can give it a shot. Just left one comment, okay to approve after that.

**Review by djahandarie — APPROVED** [review]

**Pull request merged**